cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -195,6 +195,19 @@ func (o *Options) Run(args []string) error {
 		status = status.withClusterIssuer(clusterIssuer, issuerErr)
 	}
 
+	// Nothing to output about Order and Challenge if no CR
+	if req != nil {
+		// Get Order
+		order, orderErr := findMatchingOrder(o.CMClient, ctx, req)
+		if orderErr != nil {
+			orderErr = fmt.Errorf("error when finding Order: %w\n", orderErr)
+		} else if order == nil {
+			orderErr = errors.New("No Order found for this Certificate\n")
```

Comment:
We should only output this if it is an ACME issuer

Reply:
We `do` only output it if it is an ACME issuer because I added a boolean `IsACME` to IssuerStatus and use it in the `String()` function of `CertificateStatus`.
I am working on a cleaner way to make the check without unnecessarily creating the `OrderStatus`, but that requires some refactoring around the code getting the Issuer/ClusterIssuer, which I am already working on.

Reply:
Thanks for clarification! Missed that :)